docs/File-Formats.md

```diff
@@ -724,56 +724,93 @@ AKT1 AKT2 10000|AKT<TAB>0.17071492725<TAB>0.264067254391
 ```
 
 
-## Fusion Data  
+## Structural Variation Data  
 
 #### Meta file
-The fusion metadata file should contain the following fields:
+The structural variation metadata file should contain the following fields:
 
 1. **cancer_study_identifier**: same value as specified in [study meta file](#cancer-study)
-2. **genetic_alteration_type**: FUSION
-3. **datatype**: FUSION
-4. **stable_id**: fusion
-5. **show_profile_in_analysis_tab**: true.
-6. **profile_name**: A name for the fusion data, e.g., "Fusions".
-7. **profile_description**: A description of the fusion data.
+2. **genetic_alteration_type**: STRUCTURAL_VARIATION
+3. **datatype**: STRUCTURAL_VARIATION
+4. **stable_id**: structural_variation
+5. **show_profile_in_analysis_tab**: true
+6. **profile_name**: A name for the structural variation data, e.g., "Fusions"
+7. **profile_description**: A description of the structural variation data.
 8. **data_filename**: your datafile
 9. **gene_panel (Optional)**:  gene panel stable id
 
 #### Example
 An example metadata file would be:
 ```
-cancer_study_identifier: brca_tcga_pub
-genetic_alteration_type: FUSION
-datatype: FUSION
-stable_id: fusion
+cancer_study_identifier: study_es_0
+genetic_alteration_type: STRUCTURAL_VARIATION
+datatype: SV
+data_filename: data_structural_variation.txt
+stable_id: structural_variation
+profile_name: Targeted Fusion Assay data
+profile_description: Targeted Fusion Assay data sequenced with Ion Torrent.
 show_profile_in_analysis_tab: true
-profile_name: Fusions
-profile_description: Fusion data.
-data_filename: data_fusions.txt
 ```
 
 #### Data file
-A fusion data file is a two dimensional matrix with one gene per row.  For each gene (row) in the data file, the following tab-delimited values are required in the order specified:
-
-1. **Hugo_Symbol**: A [HUGO](http://www.genenames.org/) gene symbol.
-2. **Entrez_Gene_Id**: A [Entrez Gene](http://www.ncbi.nlm.nih.gov/gene) identifier.
-3. **Center**: The sequencing center.
-4. **Tumor_Sample_Barcode**: This is the sample ID.
-5. **Fusion**: A description of the fusion, e.g., "TMPRSS2-ERG fusion".
-6. **DNA_support**: Fusion detected from DNA sequence data, "yes" or "no".
-7. **RNA_support**: Fusion detected from RNA sequence data, "yes" or "no".
-8. **Method**: Fusion detected algorithm/tool.
-9. **Frame**: "in-frame" or "frameshift".
+A structural variation data file is a two dimensional matrix with one structural variant per row. A small number of coloums is required, the others are optional.
+
+Required columns in for every type of structural variation:
+- **Sample_ID**
+- **Annotation**
+- **Event_Info**
+- **Site1_Entrez_Gene_Id** and/or **Site1_Hugo_Symbol**
+- **Site1_Exon**
+
+Additional required columns for fusion structural variation:
+- **Site2_Entrez_Gene_Id** and/or **Site2_Hugo_Symbol**
+- **Site2_Exon**
```

Comment:
exon OR position

Reply:
We changed the design: Exon should be required.